P/Pathfinder/build_tarballs.jl

```diff
@@ -0,0 +1,71 @@
+using BinaryBuilder
+
+name = "Pathfinder"
+version = v"0.5.0"
+sources = [
+    GitSource("https://github.com/servo/pathfinder.git", "7281a607a80dacc5470d7d3ec208341b6f0b0727"),
+    DirectorySource("./bundled")
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir
+cd ${WORKSPACE}/srcdir/pathfinder*
+
+# by default, pathfinder will compile to a static library,
+# so we change "staticlib" to "cdylib" to compile to a dynamic library
+atomic_patch -p1 ../patches/Compile-to-dynamic-lib.patch
+
+# we need cbindgen to generate the header file
+cargo install cbindgen --force --target x86_64-unknown-linux-musl
+
+cd c
+
+# generate the header file
+cbindgen \
+    --lang c \
+    --crate pathfinder_c \
+    --config cbindgen.toml \
+    --output "${includedir}/pathfinder.h"
+
+# build pathfinder
+cargo build --release --lib --target=${rust_target}
+
+# On windows the generated .dll doesn't start with lib
+if [[ "${target}" == *-mingw* ]]; then
+    cp \
+        ../target/${rust_target}/release/pathfinder.dll \
+        ../target/${rust_target}/release/libpathfinder.dll
+fi
+
+# install the library
+install -D -m 755 "../target/${rust_target}/release/libpathfinder.${dlext}" "${libdir}/libpathfinder.${dlext}"
+
+# install the licenses
+install_license ../LICENSE-APACHE ../LICENSE-MIT
+"""
+
+platforms = supported_platforms(; experimental=true)
```

Comment:
No need to use `experimental=true` anymore :wink: 
```suggestion
platforms = supported_platforms()
```